exercises/practice/resistor-color-expert/.meta/config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "authors": [
+    "meatball133",
+    "bethanyg"
+  ],
+  "files": {
+    "solution": [
+      "resistor_color_expert.py"
+    ],
+    "test": [
+      "resistor_color_expert_test.py"
+    ],
+    "example": [
+      ".meta/example.py"
+    ]
+  },
+  "blurb": "Convert color codes, as used on resistors with different amounts of bands, to a human-readable label."
```

Comment:
```suggestion
  "blurb": "Convert color codes as used on resistors with different bands to a human-readable label."
```

Reply:
Arg.  The review interface isn't letting me make this suggestion, but see [this comment](https://github.com/exercism/problem-specifications/pull/2291#discussion_r1231336449) from problem specifications.  We should probably note that this is adapted from Erik and Maud and use the URL referenced.